combat-trainer.lic

```diff
@@ -2153,7 +2161,9 @@ class GameState
   end
 
   def melee_attack_verb
-    if attack_override
+    if @use_tactics_combos && !offhand? && brawl_tactics? && !backstab? && tactics_analyze?
+      @tactics_combo_array.shift
```

Comment:
Since we're never using the fact that this is an array something like `@next_combo_attack` would make more sense.